### DISCUSSION TITLE
Error: Cannot find module './uws_win32_x64_83.node'

### DISCUSSION THREAD

**Discussion by vijay180396** [uNetworking/uWebSockets.js#352] [Q&A] [answered]
i am trying to connect an local server in browser

But i get a error in uWebSockets.js folder.
when i run the server it shows cannot find modules uws_win32_x64_83.node.
beacuse i found that in uWebSockets.js folder there only 
1.uws_win32_x64_64.node
2.uws_win32_x64_67.node
3.uws_win32_x64_72.node

but i cannot able to find uws_win32_x64_83.node.

And i using node version of 14.5.0.
OS=>Windows 10

**Comment by ghost** [comment, marked as answer]
Node.js 14 was not supported when you tried this
